ext/mbstring/tests/bug43994.phpt

```diff
@@ -49,7 +49,8 @@ With $regs arg:
 
 Warning: mb_ereg(): empty pattern in %s on line %d
 bool(false)
-NULL
+array(0) {
+}
```

Comment:
@cmb69 Ah I see, you want to match the `ereg` api. I didn't understand that because as far as I can remember `ereg` has been either deprecated or recommended against in favour of `preg`. So I literally _never_ used the `ereg` functions :laughing: